custom_components/plugwise/climate.py

```diff
@@ -161,13 +162,14 @@ async def async_set_hvac_mode(self, hvac_mode: str) -> None:
         )
 
         # pw-beta: feature request - mimic HomeKit behavior
-        if hvac_mode == HVAC_MODE_OFF:
-            await self.async_set_preset_mode(PRESET_AWAY)
-        if (
-            hvac_mode in [HVAC_MODE_HEAT, HVAC_MODE_COOL]
-            and self.device["active_preset"] == PRESET_AWAY
-        ):
-            await self.async_set_preset_mode(PRESET_HOME)
+        if CONF_HOMEKIT_EMULATION:
+            if hvac_mode == HVAC_MODE_OFF:
```

Comment:
Use `self._homekit_mode` for `hvac_mode` 2x, to show that this is pw-beta code, different from Core Plugwise code.

Reply:
Adjusted accordingly